sbkeys/generate-aws-sbkeys

```diff
@@ -212,10 +244,10 @@ gencert() {
   cert_arn="$(\
     aws acm-pca issue-certificate \
       --certificate-authority-arn "${ca_arn}" \
-      --template-arn arn:aws:acm-pca:::template/BlankEndEntityCertificate_APICSRPassthrough/V1 \
+      --template-arn arn:${aws_partition}:acm-pca:::template/BlankEndEntityCertificate_APICSRPassthrough/V1 \
```

Comment:
Potentially we could get this dynamically to simplify the interface:
```
aws sts get-caller-identity | jq -r '.Arn' | awk -F: '{ print $2 }'
```

Reply:
Yeah thats a great idea!